task-sdk/src/airflow/sdk/definitions/deadline.py

```diff
@@ -124,6 +124,28 @@ def __init__(self, callback_callable: Callable | str, kwargs: dict | None = None
         self.path = self.get_callback_path(callback_callable)
         self.kwargs = kwargs
 
+    @staticmethod
+    def _is_valid_dotpath(path: str) -> bool:
```

Comment:
this method on deadline class feels weird!

Reply:
Hmm yeah, what would you think of doing it at module level instead?

Reply:
Yeah, anywhere in a "utils" type location would be a fair call IMHO.  I didn't think of `module_loading.py` but I like that solution.